deployment/network-operator/README.md

```diff
@@ -135,7 +174,7 @@ of the network operator in the following manner:
 
 #### Example 1
 Network Operator deployment with a specific version of OFED driver and a single RDMA resource mapped to `enp1`
-netdev.
+netdev, also deploying secondary network with Mutlus CNI, containernetworking-plugins CNI plugins and whereabouts IPAM CNI Plugin.
```

Comment:
lets have this as example 3 which focuses on secondary network enablement with RDMA

i.e
deployment of device plugin and secondary network components.

device selectors PR can then be integrated into it to have a simple recipe to enable RDMA on secondary network.